src/qt/guiutil.h

```diff
@@ -68,14 +68,22 @@ namespace GUIUtil
        @param[in] role    Data role to extract from the model
        @see  TransactionView::copyLabel, TransactionView::copyAmount, TransactionView::copyAddress
      */
-    void copyEntryData(QAbstractItemView *view, int column, int role=Qt::EditRole);
+    void copyEntryData(const QAbstractItemView *view, int column, int role=Qt::EditRole);
 
     /** Return a field of the currently selected entry as a QString. Does nothing if nothing
         is selected.
        @param[in] column  Data column to extract from the model
        @see  TransactionView::copyLabel, TransactionView::copyAmount, TransactionView::copyAddress
      */
-    QList<QModelIndex> getEntryData(QAbstractItemView *view, int column);
+    QList<QModelIndex> getEntryData(const QAbstractItemView *view, int column);
+
+    /** Check whether a field of the currently selected entry of a view is not empty. Does nothing if
+        nothing is selected.
```

Comment:
This doc is for a boolean, so remove "Does nothing if nothing is selected."

Suggested change (or something similar/simpler):
```diff
-    /** Check whether a field of the currently selected entry of a view is not empty. Does nothing if
-        nothing is selected.
+    /** Returns true if the specified field of the currently selected view entry is not empty.
```